test/conformance/source/environment.cpp

```diff
@@ -281,7 +281,7 @@ DevicesEnvironment::DevicesEnvironment(int argc, char **argv)
             error = "urDeviceGet() failed to get devices.";
             return;
         }
-        for (u_long i = 0; i < count; i++) {
+        for (unsigned i = 0; i < count; i++) {
```

Comment:
any reason just a regular `int` can't be used?

Reply:
Using a signed `int` triggers warnings C4018 and `-Wsign-compare`.

Reply:
makes sense.